build/lib/builtInExtensions.js

```diff
@@ -18,7 +18,9 @@ const fancyLog = require('fancy-log');
 const ansiColors = require('ansi-colors');
 
 const root = path.dirname(path.dirname(__dirname));
-const builtInExtensions = require('../builtInExtensions.json');
+// {{SQL CARBON EDIT}}
+const builtInExtensions = require('../builtInExtensions-insiders.json');
```

Comment:
This file is only called from sql.sh and sql.bat, hence I am changing to always use the "insiders" stream. In the actual build we choose based on product quality instead.